docs/topics/mta-web-config-maven-rep.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-config-maven-rep_{context}"]
+= Configuring Maven repositories
+
+To configure a Maven repository or to clear the local artifact repository proceed with the following steps:
+
+== Configuring the repository
+
+.Procedure
+
+. In the left menu of the *Administrator* view, click *Repositories* and then *Maven*.
+. Click *Force update of dependencies* toggle switch to enable.
```

Comment:
I think this option has been removed. We need confirmation from @jortel

Reply:
@rromannissen waiting for a response.

Reply:
Yes. removed.